.github/CODEOWNERS

```
@@ -0,0 +1,7 @@
+# CODEOWNERS: https://help.github.com/articles/about-codeowners/
+
+# Docs 
+*.md @citmc @cmwaters @nooomski @coldice 
+
+# Primary repo maintainers
+* @coldice @gioelebonifetto @8bitpal 
```

```suggestion
* @coldice @xavierlepretre @gioelebonifetto @8bitpal 
```
Also here, adding Xavier

---

@okwme wdyt, do we need someone from IG/AiB on this list as well?

---

@toschdev and I struggled for so long as the only two approvers... 
hi @okwme !! 
CODEOWNERS assigns the reviewer automatically. You can choose to add reviewers at will to any PR.

---

Yeah ok cool let's just go ahead with this and iterate when needed.